storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -6182,6 +6216,31 @@ static bool xtrabackup_close_temp_log(bool clear_flag) {
   srv_log_group_home_dir = srv_log_group_home_dir_save;
   innobase_log_file_size = innobase_log_file_size_save;
 
+  /* remove #innodb_redo folder */
+  if (!xtrabackup_incremental_dir) {
+    sprintf(redo_folder, "%s/%s", xtrabackup_target_dir, LOG_DIRECTORY_NAME);
+  } else {
+    sprintf(redo_folder, "%s/%s", xtrabackup_incremental_dir,
+            LOG_DIRECTORY_NAME);
+  }
+
+  /* remove #innodb_redo dir if exist */
+  if (!os_file_scan_directory(
+          redo_folder,
+          [](const char *path, const char *file_name) {
+            if (strcmp(file_name, ".") == 0 || strcmp(file_name, "..") == 0) {
+              return;
+            }
+            const auto to_remove =
+                std::string{path} + OS_PATH_SEPARATOR + file_name;
+            unlink(to_remove.c_str());
+          },
+          true)) {
+    ib::error(ER_IB_CLONE_STATUS_FILE)
```

Comment:
Better to not use CLONE_STAUTS_FILE tag?

Reply:
not present in the latest commit.